vispy/geometry/curves.py

```diff
@@ -0,0 +1,343 @@
+# ----------------------------------------------------------------------------
+#  Anti-Grain Geometry (AGG) - Version 2.5
+#  A high quality rendering engine for C++
+#  Copyright (C) 2002-2006 Maxim Shemanarev
+#  Contact: mcseem@antigrain.com
+#           mcseemagg@yahoo.com
+#           http://antigrain.com
+#
+#  AGG is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
```

Comment:
It looks like there is also a reference to GNU GPL licensing in `svg/geometry.py`. @rougier do we have permission for that one, too? If not we have to remove it.